src/app/story/directives/podcast.component.ts

```diff
@@ -33,12 +37,53 @@ export class PodcastComponent implements OnDestroy {
 
   story: StoryModel;
   tabSub: Subscription;
-  itunesExplicitDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesRequirementsDoc = 'https://help.apple.com/itc/podcasts_connect/#/itc1723472cb';
+  itunesExplicitDoc = 'https://support.apple.com/en-us/HT202005';
 
-  OPTIONS = ['Explicit', 'Clean'];
+  OPTIONS = ['', 'Explicit', 'Clean'];
 
-  constructor(tab: TabService) {
-    this.tabSub = tab.model.subscribe((s: StoryModel) => this.story = s);
+  constructor(tab: TabService,
+              private cms: CmsService) {
+    this.tabSub = tab.model.subscribe((s: StoryModel) => this.init(s));
+  }
+
+  init(story) {
+    this.story = story;
+    this.getSeriesOverrides();
+  }
+
+  getSeriesOverrides() {
+    this.cms.account.subscribe(account => {
+      if (this.story.isNew && this.story.parent && this.story.parent.isa('series')) {
+        this.defaultDistributionOverrides(new SeriesModel(account, this.story.parent, false));
+      }
+    });
+  }
+
+  defaultDistributionOverrides(series: SeriesModel) {
+    series.loadRelated('distributions').subscribe(distributions => {
+      this.defaultExplicitToSeries(<DistributionModel[]>distributions);
+    });
+  }
+
+  defaultExplicitToSeries(distributions: DistributionModel[]) {
+    let dist = distributions.find(d => d.kind === 'podcast');
+    dist.loadRelated('versionTemplate').subscribe(() => {
+      dist.loadExternal().subscribe(() => {
+        this.story.loadRelated('versions').subscribe(() => {
+          this.story.versions.forEach(v => {
+            // well crap, new story's version doc is null
```

Comment:
Okay, so this is actually really close.  (I ended up going down some rabbit holes debugging, because I had some pre-this-PR distributions that didn't have any audio_version_template set).

So the `VersionModel` may not have a doc, but it does have a `template` property, which will be non-null if it was init'd with an `AudioVersionTemplateModel`.  So you can just check `if (v.template && v.template.id === dist.versionTemplate.id)` or something.

AudioVersionModel is a bit strange in that its AudioVersionTemplateModel isn't actually related to it.  It's assigned to it from the constructor, because new AudioVersionModels need to know which template they are for right away.